src/effective-dart/usage.md

```diff
@@ -380,7 +385,7 @@ class UploadException {
 ```
 
 Assigning to a local variable can be cleaner and safer than using `!` every
-place the field or top-level variable is used:
+place the instance is used:
```

Comment:
It took me a while to understand what "the instance" was referring to, due to the technical meaning of "instance" I mentioned earlier. If you follow my suggestion above and change "instance" to "value", then this sentence says "...safer than using `!` every place the value is used", which I think will be easier to follow.